motr/client.h

```diff
@@ -596,28 +596,34 @@ enum m0_entity_type {
  */
  enum m0_entity_flags {
 	/**
-	 * During create if this flag is set in entity->en_flags, that means
-	 * application has capability to store meta-data and hence pver and
-	 * lid can be stored in  application's meta-data.
-	 * Before calling to m0_entity_create/open(), application is
-	 * expected to set obj>ob_entity->en_flags |= M0_ENF_META, so when
-	 * m0_entity_create() returns to application, pool version and layout id
-	 * will be available to application into obj->ob_attr.oa_pver and
-	 * obj->ob_attr.oa_lid respectively and can be stored into application's
-	 * meta-data.
+	 * If motr client application has the capability to store object
+	 * metadata (like pver and lid) by itself (for example, at motr
+	 * distributed index), it can use this flag to inform libmotr about
+	 * this and avoid sending additional metadata fops on such object
+	 * operations as CREATE, OPEN, DELETE, GETATTR and, thus, improve
+	 * its performance.
+	 *
+	 * Before calling m0_entity_create() or m0_entity_open(), application
+	 * is expected to set obj->ob_entity->en_flags |= M0_ENF_META. When
+	 * m0_entity_create() returns, the pool version and layout id will be
```

Comment:
s/layout id/layout type

Reply:
It's not type. It's id. Layout type is the same - parity declustering.